examples/editor-core/overlays/example.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useEffect, useState} from 'react';
+import DeckGL from '@deck.gl/react';
+import StaticMap from 'react-map-gl';
+import {INITIAL_COORDS, INITIAL_VIEW_STATE, MAPBOX_ACCESS_TOKEN} from './constants';
+import {HtmlOverlay, HtmlOverlayItem} from '@deck.gl-community/react-editable-layers';
```

Comment:
It looks like these exports are missing from the react-editable-layers` `index.js` entrypoint, does that need to be included in this PR as well?

Reply:
My bad, I did not push those changes :/

Reply:
done